apps/webstore/src/app/shared/components/products/pet-sign-design/pet-sign-design.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="design-wrapper" #designWrapper [ngStyle]="{ 'font-family': this.design.font }">
+    <div class="above-canvas" *ngIf="!isLoading">
+        <div
+            #inputWrapper
+            id="input-wrapper"
+            [ngClass]="{ 'show-dashed-line': showInputFieldOptions && isMutable }"
+            [ngStyle]="{ width: inputWidth + '%' }"
+        >
+            <textarea
+                matInput
+                id="title-input"
+                cdkTextareaAutosize
+                #titleInput
+                #autosizeTitle="cdkTextareaAutosize"
+                cdkAutosizeMinRows="1"
+                cdkAutosizeMaxRows="25"
+                spellcheck="false"
+                [ngStyle]="{ 'font-size': fontSize * 2 + 'px' }"
+                [maxlength]="40"
```

Comment:
I dont think we need to add a maxlength. If the user wants a long ass name and no subname, it should be possible to add more than 40 chars :)

Reply:
Resolved in [08f856f](https://github.com/treecreate/webstore/pull/406/commits/08f856f85c803fb32bbc0807732d032cab109118)